ScratchWikiSkinTemplate.php

```diff
@@ -161,6 +161,7 @@ public function execute() {
 <?php foreach ( $this->getFooterLinks('flat') as $key ) { ?>
 					<li><?php $this->html( $key ) ?></li>
 <?php } ?>
+					<?php if(!empty($wgRightsIcon)){ ?><br><a href="<?php if(!empty($wgRightsPage)){echo Title::newFromText($wgRightsPage)->getLocalURL();}else{echo $wgRightsUrl;} ?>"><img src="<?php echo $wgRightsIcon ?>"></a><?php } ?>
```

Comment:
I see how it works, but could you make it not a one-liner for readability? So change this to:
```php
<?php if (!empty( $wgRightsIcon )) { ?>
<br/>
<a href="<?php if (!empty( $wgRightsPage)) {
	echo Title::newFromText($wgRightsPage)->getLocalURL();
} else {
	echo $wgRightsUrl;
}?>">
<img src="<?=$wgRightsIcon?>" />
</a>
<?php } ?>
```

Reply:
Ok